### ISSUE TITLE
[Portal] Valdrakken to Stormwind

### ISSUE THREAD

**Issue by Scarlet-Dragon** [Dathwada/handynotes-travelguide#22]
The portal from Valdrakken (Seat of the Aspects) to Stormwind shows a blue colour indicating that it is available, but there appears to be a pre-req so should show in red with details of Unlocked with Quest for those who haven't done the required quest.

v2.14.0

![image](https://user-images.githubusercontent.com/54205922/208463713-3c72215e-0411-43c8-8ad1-af22034cd57c.png)

**Comment by Dathwada** [comment]
Hi, I'm aware of this but I haven't figured out what this requirement is.

**Comment by Dathwada** [comment]
It might be this Quest: [Moving On](https://www.wowhead.com/quest=66221/moving-on).

**Comment by Scarlet-Dragon** [comment]
> It might be this Quest: [Moving On](https://www.wowhead.com/quest=66221/moving-on).

Could be, /run print(C_QuestLog.IsQuestFlaggedCompleted(66221)) = false 
Will keep an eye on my campaign, and see if the portal becomes available once I complete it.

**Comment by Scarlet-Dragon** [comment]
> It might be this Quest: [Moving On](https://www.wowhead.com/quest=66221/moving-on).

No, this quest isn't the trigger. I've not completed that quest, but I now see the portal. Since last time I checked (without portal), I have two new achievements, Azure Spanner and Explore the Azure Span. and sis the campaign quests up to The Blue Dragon Oathstone.
